### PR TITLE
log in json to be indexable

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/compute 

```
{"severity":"INFO","message":"Connecting to vault","address":"FOO.com:8200","v2":true}
{"severity":"INFO","message":"secrets found","keys":[["MYSQL_PASSWORD","foo/kube_stats/pod-105/mysql_password"],["MYSQL_USER","foo/kube_stats/pod-105/mysql_user"]]}
{"severity":"INFO","message":"secrets written","keys":["MYSQL_PASSWORD","MYSQL_USER","ROLLBAR_ACCESS_TOKEN"]}
```

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -37,9 +37,3 @@ desc "Run tests in docker"
 task test_in_docker: :build_dev do
   sh "docker run -it --rm zendesk/samson_secret_puller-dev rake"
 end
-
-desc "Release a new version"
```
Comment (adammw) [inline Rakefile]: why are we getting rid of this?

**Comment by adammw** [comment]
Can you make `present_errors` also produce / raise JSON (or have a global exception handler to print JSON)?

**Comment by grosser** [comment]
added json logging for exceptions too

File: Rakefile (hunk 1 above)
Comment (grosser) [reply Rakefile]: should not be used anymore since we have auto-build, see Readme

**Pull request merged**